e2e/paths/uploadDocuments_test.js

```diff
@@ -7,19 +7,25 @@ Before((I, caseViewPage) => {
   caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
 });
 
-Scenario('Selecting to follow for a document produces a textarea', (I, uploadDocumentsPage) => {
-  I.selectOption(uploadDocumentsPage.fields.socialWorkChronologyStatus, 'To follow');
-  I.fillField(uploadDocumentsPage.fields.socialWorkChronologyReason, 'mock reason');
+Scenario('Completing half the Annex documents form by selecting social work chronology document to follow in the c110a application', (I, uploadDocumentsPage, caseViewPage) => {
+  uploadDocumentsPage.socialWorkChronologyToFollow();
   I.continueAndSubmit(config.eventSummary, config.eventDescription);
   I.seeEventSubmissionConfirmation(config.applicationActions.uploadDocuments);
+  caseViewPage.selectTab('evidence');
+  I.seeDocument('Social work chronology', '', 'To follow', 'mock reason');
 });
 
-Scenario('All documents are able to be uploaded', (I, uploadDocumentsPage) => {
-  uploadDocumentsPage.uploadSocialWorkChronology(config.testFile);
+Scenario('Complete the Annex documents part of the form by uploading all files in the c110a application', (I, uploadDocumentsPage, caseViewPage) => {
+  uploadDocumentsPage.socialWorkChronologyToFollow(config.testFile);
   uploadDocumentsPage.uploadSocialWorkStatement(config.testFile);
   uploadDocumentsPage.uploadSocialWorkAssessment(config.testFile);
   uploadDocumentsPage.uploadCarePlan(config.testFile);
   uploadDocumentsPage.uploadAdditionalDocuments(config.testFile);
   I.continueAndSubmit(config.eventSummary, config.eventDescription);
   I.seeEventSubmissionConfirmation(config.applicationActions.uploadDocuments);
+  caseViewPage.selectTab('evidence');
```

Comment:
Why not to do `tabs.evidence`? Using text that must match property name makes it fragile.